WordPress/Classes/ViewRelated/Blog/QuickStartChecklistHeader.swift

```diff
@@ -15,11 +16,29 @@ class QuickStartChecklistHeader: UIView {
                 self?.bottomStroke.alpha = CGFloat(alpha)
                 self?.chevronView.transform = CGAffineTransform(rotationAngle: rotate)
             })
+
+            let hintToken: String
+            if collapse {
+                hintToken = NSLocalizedString("Collapses", comment: "This token is used with ~ to format the accessibility hint to reflect the expanded and collapsed state of the list.")
+            } else {
+                hintToken = NSLocalizedString("Expands", comment: "This token is used with ~ to format the accessibility hint to reflect the expanded and collapsed state of the list.")
+            }
+
+            let accessibilityFormat = NSLocalizedString("%@ the list of completed tasks.", comment: "Accessibility hint for the list of completed tasks presented during Quick Start.")
```

Comment:
In this case I believe that is better to localize the whole sentence. In other languages the order of words might be different, and the "Expands"/"Collapses" translation won't necessarily be always at the beginning.